test/unit/controlbar-test.js

```diff
@@ -14,6 +14,10 @@ describe('Control Bar', function() {
     model.mediaController = {};
     model.mediaController.on = sinon.stub();
     model.mediaController.on.returnsThis();
+    const playerModel = new SimpleModel();
+    playerModel.change = sinon.stub();
+    playerModel.change.returnsThis();
+    model.player = playerModel;
```

Comment:
this is still needed, as `SimpleModel` does not have an instance of player model.

Reply:
👍